dev/bin/commands.sh

```diff
@@ -37,15 +38,27 @@ wp_audit() {
 
 # Set up WordPress unit testing environment
 wp_setup_tests() {
-    docker exec -it dev-wordpress-php-fpm-1 /bin/sh -c "/usr/bin/setup-tests.sh wordpress_test root ${MYSQL_ROOT_PASSWORD} ${MYSQL_HOST}"
+    CLI_CONTAINER=$(docker ps --filter "ancestor=wordpress:cli-php7.4" --format "{{.Names}}")
+    CURRENT_WORDPRESS_VERSION=$(docker exec -it $CLI_CONTAINER wp core version | tr -d '\r')
+    echo "Current WordPress Version `${CURRENT_WORDPRESS_VERSION}` on container ${CLI_CONTAINER}" 
+    docker exec -it \
+    $(docker ps --filter "ancestor=des-wordpress-php-fpm:latest" --format "{{.Names}}") \
+    /bin/sh -c \
+    "/usr/bin/setup-tests.sh \
+    wordpress_test \
+    root \
+    ${MYSQL_ROOT_PASSWORD} \
+    ${MYSQL_HOST} \
+    ${CURRENT_WORDPRESS_VERSION}"
```

Comment:
I'm getting this output when I run this:
```sh
% wp_setup_tests
wp_setup_tests:3: command not found: 6.1.1
Current WordPress Version  on container bcfd-wp-cli-1
```
But it seems to be installing the 6.1.1 test files so I think it's just the echoing that's not working correctly.